### PR TITLE
Add API documentation

### PR DESCRIPTION
**Pull request by xdumaine**
Opening this preemptively to get feedback on format, etc before it's all the way complete. Still working on fleshing it out.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Want to see it in action? Check out the demo: https://talky.io/
 <!DOCTYPE html>
 <html>
     <head>
-        <script src="http://simplewebrtc.com/latest-v2.js"></script> 
+        <script src="http://simplewebrtc.com/latest-v2.js"></script>
```
Comment (fippo) [inline README.md]: src="https://..."

File: README.md [hunk 2]
```diff
@@ -82,3 +82,102 @@ or the Gitter channel:
...
+
+`getLocalScreen()`
+
+`stopScreenShare()`
+
+`testReadiness()`
+
+`createRoom(name, callback)`
+
+`sendFile()`
```
Comment (fippo) [inline README.md]: let's not document this :-)

File: README.md [hunk 3]
```diff
@@ -82,3 +82,102 @@ or the Gitter channel:
...
+
+### Connection
+
+By default, SimpleWebRTC uses a Socket.io connection to communicate with the signaling server. However, you can provide an alternate connection object to use. All that your alternate connection need provide are four methods:
```
Comment (fippo) [inline README.md]: lowercase socket.io (I think?)

**Comment by fippo** [comment]
LGTM (apart from nits). Thanks!

File: README.md [hunk 4]
```diff
@@ -82,3 +82,102 @@ or the Gitter channel:
...
 
 ## API
+
+### Constructor
+
+`new SimpleWebRTC(options)`
+
+- `object options` - options object provided to constructor consisting of:
+  - `string url` - *required* url for signaling server. Defaults to signaling server URL which can be used for development. You must use your own signaling server for production.
+  - `object sockio` - *optional* object to be passed as options to the signaling server connection.
```
Comment (tgabi333) [inline README.md]: isn't it socketio?  see: https://github.com/andyet/SimpleWebRTC/blob/master/socketioconnection.js#L4

Comment (xdumaine) [inline README.md]: Yep, just a typo. Fixed. Thanks!

**Pull request merged**